modules/oe_webtools_analytics/modules/oe_webtools_analytics_rules/oe_webtools_analytics_rules.install

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * @file
+ * Contains install and update functions of OpenEuropa Webtools Analytics Rules.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Cache\Cache;
+
+/**
+ * Implements hook_install().
+ */
+function oe_webtools_analytics_rules_install() {
+  // Because the OpenEuropa Webtools Analytics Rules module contains
+  // event subscriber which use cache context 'url.path', we must invalidate
+  // all cached HTML output.
+  Cache::invalidateTags(['rendered']);
+  \Drupal::service('twig')->invalidate();
```

Comment:
I'm not sure about this, we don't clear caches in `hook_install()`s. Is this module enabled optionally by Behat by any chance, when we run the test?

Reply:
Yes. for oe_webtools we are doing enabling modules per Behat feature.
About clearing cache on install, in core some modules also doing a clearing of render cache on install like here: https://github.com/drupal/core/blob/8.8.x/modules/block/block.install#L17 and here: https://github.com/drupal/core/blob/8.8.x/modules/layout_builder/layout_builder.install#L40.